public/app/plugins/datasource/azuremonitor/components/MonitorConfig.tsx

```diff
@@ -1,19 +1,14 @@
-import React, { useMemo, useState } from 'react';
+import React, { useMemo, useEffect, useState } from 'react';
 
 import { SelectableValue } from '@grafana/data';
 import { config } from '@grafana/runtime';
 
-import { getCredentials, updateCredentials } from '../credentials';
-import { AzureDataSourceSettings, AzureCredentials } from '../types';
-
+import { AzureDataSourceSettings } from '../types';
+import { AzureCredentials } from './AzureCredentials';
+import { getCredentials, getDefaultCredentials, hasCredentials, updateCredentials } from './AzureCredentialsConfig';
 import { AzureCredentialsForm } from './AzureCredentialsForm';
 import { DefaultSubscription } from './DefaultSubscription';
-
-const azureClouds: SelectableValue[] = [
-  { value: 'azuremonitor', label: 'Azure' },
-  { value: 'govazuremonitor', label: 'Azure US Government' },
-  { value: 'chinaazuremonitor', label: 'Azure China' },
-];
+import {KnownAzureClouds} from "../../prometheus/configuration/AzureCredentials";
```

Comment:
This needs to be imported from `./AzureCredentials.ts`

Reply:
Yeah thanks, seems like automatic refactoring messed up.